libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/client/EvcsClient.java

```diff
@@ -156,14 +167,27 @@ private URI getUri(String userId, List<EvcsVCState> vcStatesToQueryFor)
     private void checkResponseStatusCode(HttpResponse<String> evcsHttpResponse)
             throws EvcsServiceException {
         if (200 > evcsHttpResponse.statusCode() || evcsHttpResponse.statusCode() > 299) {
+            String responseMessage;
+            try {
+                Map<String, String> responseBody =
+                        OBJECT_MAPPER.readValue(evcsHttpResponse.body(), new TypeReference<>() {});
```

Comment:
If we do anything more complex than this, we should create a proper `EvcsErrorResponse` class, but this is fine for now.